### PR TITLE
Caesar cipher

### PR DESCRIPTION
**Pull request by Karthik-B-06**
Added Caesar Cipher Encryption #7

### CODE REVIEW

File: caesar-cipher/caesar-cipher.c [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include <math.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <limits.h>
+#include <stdbool.h>
```
Comment (saurabh47) [inline caesar-cipher/caesar-cipher.c:7]: remove redundant includes

Comment (Karthik-B-06) [reply caesar-cipher/caesar-cipher.c]: Fixed.

File: caesar-cipher/caesar-cipher.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+    int n; 
+    scanf("%d",&n);
```
Comment (saurabh47) [inline caesar-cipher/caesar-cipher.c:6]: Please add print statements before taking Input & for Output
For Example:
```
Enter Input String Length: 3
Enter Input String: abc
Enter Key: 1
Output : bcd
```

Comment (Karthik-B-06) [reply caesar-cipher/caesar-cipher.c]: Fixed it.

File: caesar-cipher/caesar-cipher.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+    int n; 
+    scanf("Enter Input String Length: %d",&n);
```
Comment (saurabh47) [inline caesar-cipher/caesar-cipher.c:6]: Please check your code.you can print statement while reading in c language.
Please 
https://stackoverflow.com/questions/12445474/how-to-print-using-scanf-function-in-c-programming

Comment (Karthik-B-06) [reply caesar-cipher/caesar-cipher.c:6]: Oops. My bad.

**Review by saurabh47 — APPROVED** [review]

**Pull request merged**